src/build.props

```diff
@@ -11,7 +11,7 @@
     <Product Condition=" '$(Product)' == '' ">Microsoft SARIF SDK</Product>
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
     
-    <!-- VersionPrefix denotes the current Semantic Version for the SDK. Must be updated before every nuget drop. -->
```

Comment:
>Semantic Version [](start = 43, length = 16)

While I'm in here: We can't claim SemVer compatibility because we make breaking changes in patch-level versions.